Detectors/FIT/T0/simulation/include/T0Simulation/DigitizerTask.h

```diff
@@ -54,10 +54,10 @@ class DigitizerTask : public FairTask
   Bool_t mContinuous = kFALSE;  ///< flag to do continuous simulation
   double mFairTimeUnitInNS = 1; ///< Fair time unit in ns
 
-  Int_t mSourceID = 0;                                       ///< current source
-  Int_t mEventID = 0;                                        ///< current event id from the source
-  Digitizer mDigitizer;                                      ///< Digitizer
-  const std::vector<o2::fit::HitType>* mHitsArray = nullptr; ///< Array of MC hits
+  Int_t mSourceID = 0;                                      ///< current source
+  Int_t mEventID = 0;                                       ///< current event id from the source
+  Digitizer mDigitizer;                                     ///< Digitizer
+  const std::vector<o2::t0::HitType>* mHitsArray = nullptr; ///< Array of MC hits
```

Comment:
in the HitType.h the HitType is defined in the ``o2::namespace``, as well as in many places below.